code/game/objects/effects/effects.dm

```diff
@@ -39,3 +39,6 @@
 
 /obj/effect/attack_hulk(mob/living/carbon/human/user, does_attack_animation = 0)
 	return 0
+
+/obj/effect/experience_pressure_difference()
+	return
```

Comment:
Everytime I see a proc override like this, I feel like it could be done with a flag. But we only have so many